spec/controllers/blog_posts_controller_spec.rb

```diff
@@ -28,23 +28,34 @@
     let(:tag) { "cool tag" }
     let(:blog_post) { double("BlogPost") }
 
-    before :each do
-      allow(BlogPost).to receive(:tagged_with).with(tag, :on => :tags).and_return([blog_post])
+    context "tag doesn't exist" do
+      it "renders a 404 if the tag can't be found" do
```

Comment:
well.. you should test that it renders... you're actually testing that it raises an error... your decription doesn't align with the test